docs/streams/subscribe-streams.md

```diff
@@ -0,0 +1,268 @@
+---
+sidebar_position: 70
+title: Subscribe to Streams
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+This page explains how to subscribe to a stream in Macrometa.
+
+<Tabs groupId="operating-systems">
+<TabItem value="py" label="Python SDK">
+
+You must [Install the Python SDK](../sdks/install-sdks.md) before you can run this code.
+
+```py
+import base64
+import json
+from c8 import C8Client
+
+# Connect to GDN.
+URL = "gdn.paas.macrometa.io"
+GEO_FABRIC = "_system"
+API_KEY = "xxxxxx" # Change this to your API key
+is_local = False # For a global stream pass False and True for local stream
+demo_stream = "streamQuickstart"
+
+client = C8Client(protocol='https', host=URL, port=443, apikey=API_KEY, geofabric=GEO_FABRIC)
+
+# Create the subscriber and receive data
+subscriber = client.subscribe(stream=demo_stream, local=is_local,
+    subscription_name="test-subscription-1")
+for i in range(10):
+    print("In ",i)
+    m1 = json.loads(subscriber.recv())  # Listen on stream for any receiving messages
+    msg1 = base64.b64decode(m1["payload"]).decode('utf-8')
+    print(F"Received message '{msg1}' id='{m1['messageId']}'") # Print the received message
+    subscriber.send(json.dumps({'messageId': m1['messageId']})) # Acknowledge the received message
+```
+
+</TabItem>
+<TabItem value="js" label="JavaScript SDK">
+
+You must [Install the JavaScript SDK](../sdks/install-sdks.md) before you can run this code.
+
+```js
+const jsc8 = require("jsc8");
+const client = new jsc8({ url: "https://gdn.paas.macrometa.io", apiKey: "xxxxx", fabricName: "_system" });
+console.log("Authentication done!!...");
+const stream = "streamQuickstart";
+
+(async function () {
+  // Here the last boolean value tells if the stream is local or global. false means that it is global.
+  const consumer = await client.createStreamReader(stream, "my-subscription", false);
+  consumer.on("message", (msg) => {
+    const { payload, messageId } = JSON.parse(msg);
+
+    // Received message payload
+    console.log(Buffer.from(payload, "base64").toString("ascii"));
+    // Send message acknowledgement
+    consumer.send(JSON.stringify({ messageId }));
+  });
+})();
+```
+
+</TabItem>
+<TabItem value="api-py" label="REST API - Python">
+
+Use our interactive API reference with code generation in 18 programming languages to [publish a message](https://macrometa.com/docs/api#/operations/PublishStreamMessage).
```

Comment:
Is the publish a message reference intentional here meaning.. telling the user to use that PublishSTreamMessage api and then test our subscribe code...
Or I am overthinking and this was just a mistake since the page is subscribe stream? (Also this thing is repeated in the file so check all the instances)

Reply:
No, it's a copy/paste error, I must have been tired. Could have sworn I updated all those, but obviously not. ARGH. I will fix this.